cookbooks/aws-parallelcluster-config/resources/manage_ebs.rb

```diff
@@ -131,24 +131,23 @@
 
   # Mount each volume
   dev_path = [] # device labels
-  dev_uuids = [] # device uuids
 
   vol_array.each_with_index do |volumeid, index|
     dev_path[index] = "/dev/disk/by-ebs-volumeid/#{volumeid}"
-    dev_uuids[index] = get_uuid(dev_path[index])
 
     # Unmount and remove volume from /etc/fstab
-    mount shared_dir_array[index] do
-      device(DelayedEvaluator.new { dev_uuids[index] })
-      fstype(DelayedEvaluator.new { node['cluster']['volume_fs_type'] })
-      device_type :uuid
-      options "_netdev"
-      pass 0
-      action %i(unmount disable)
+    execute 'unmount ebs' do
```

Comment:
instead of the `umount` command, is it possible to keep the chef `mount action :unmount` with the mount point instead of using the UUID?

Reply:
device is required for chef mount resource, it can only be  label or uuid

Reply:
Tested use device, it doesn't work for existing ebs. Changed to get UUID from lsblk output when unmounting EBS